src/views-components/advanced-tab-dialog/metadataTab.tsx

```diff
@@ -25,7 +25,7 @@ interface MetadataTable {
 
 interface MetadataProps {
     items: MetadataTable[];
-    user: UserResource;
+    tail: any;
```

Comment:
Set correct type - do not use any, in this place you could use a render prop for example o create a component that retrieves data based on uuid.